zbench.zig

```diff
@@ -219,46 +225,162 @@ pub const Benchmark = struct {
 /// It takes a pointer to a Benchmark object.
 pub const BenchFunc = fn (*Benchmark) void;
 
-/// BenchmarkResult stores the result of a single benchmark.
-/// It includes the name and the total duration of the benchmark.
+/// BenchmarkResult stores the resulting computed metrics/statistics from a benchmark
 pub const BenchmarkResult = struct {
-    /// Name of the benchmark.
+    const Self = @This();
+    const Color = c.Color;
+
+    /// Name of the benchmark
     name: []const u8,
-    /// Total duration of the benchmark in nanoseconds.
-    duration: u64,
+    /// 75th, 99th and 99.5th percentiles of the recorded durations. They represent the timings below
+    /// which 75%, 99% and 99.5% of the other measurments would lie, respectively, when timings
+    /// are sorted in increasing order.
+    percs: Benchmark.Percentiles,
```

Comment:
I would suggest do not use the abbv. here since, this does not seem common. 
```suggestion
    percentiles: Benchmark.Percentiles,
```